sequelizeConfig.ts

```diff
@@ -0,0 +1,11 @@
+import { Sequelize, Dialect } from 'sequelize';
+import config from './config/config.json';
+
+const { username, password, database, host, dialect } = config.development;
+
+const sequelize = new Sequelize(database, username, password, {
+  host,
+  dialect: dialect as Dialect,
+});
+
+export default sequelize;
```

Comment:
Please consider using a .sequelizerc file for sequelize configuration in the root directory